src/panels/Menu/TaskBar/TaskBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Flex } from '@mantine/core';
+import { WindowLayoutOptions } from 'src/windowmanagement/WindowLayout/WindowLayout';
+
+import { menuItemsData } from '@/windowmanagement/data/MenuItems';
+
+import { TaskBarMenuButton } from './TaskBarMenuButton';
+
+interface Props {
+  addWindow: (component: React.JSX.Element, options: WindowLayoutOptions) => void;
+  visibleMenuItems: string[];
+}
+
+export function TaskBar({ addWindow, visibleMenuItems }: Props) {
+  const checkedMenuItems = menuItemsData.filter((item) => {
+    return visibleMenuItems.includes(item.componentID);
+  });
+
+  return (
+    <Flex
+      gap={2}
+      style={{
+        backgroundColor: '#00000080',
```

Comment:
Consider removing this background color.  It doesn't look great with the OS rendering in the background
![image](https://github.com/user-attachments/assets/00cff7d8-af23-405c-8a57-c1b8a3cf75ad)


Also, we should avoid using HEX colors and instead use Mantine's variables

Reply:
Actually, please just remove it :) 

![Uploading image.png…]()

Reply:
One thing to consider is that it will be unintuitive why you cannot put a panel at the very bottom, for example on the right hand side. And I think it makes the panel look weird that they don't go all the way to the bottom. That's the only reason I added the tint at all

Reply:
see eg
![image](https://github.com/user-attachments/assets/c2adf90a-d32f-46f3-af23-1f5c7d1e8ebf)

Reply:
I think it is better to not have the color as well. I agree that it looks funny with the gap but it is better than with the color imo